mirrord/config/src/feature/network/incoming.rs

```diff
@@ -397,10 +404,11 @@ impl FromStr for IncomingMode {
     fn from_str(val: &str) -> Result<Self, Self::Err> {
         match val.parse::<bool>() {
             Ok(true) => Ok(Self::Steal),
-            Ok(false) => Ok(Self::Mirror),
+            Ok(false) => Ok(Self::Off),
```

Comment:
When would this be used? Couldn't it possibly break existing configurations?

Reply:
If someone had `"incoming": false` it'll change the behavior but I believe it makes more sense.

Reply:
What if they don't have a config file, or they have a config file without the `incoming` section?

Reply:
It still mirrors (it's considered disabled config, which still defaults to mirror)